Program.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System;
+
+namespace HelloWorld
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            Console.WriteLine("Hello World!");
```

Comment:
if you replace ConsoleWriteLine with System.Diagnostics.Debug.WriteLine,
 you'd be able to see logs in Immediate Window in VS.
 
 to enable this feature go to VS->Tools->Options->Debugging->General->check 'redirect all output window text to the immediate window'